lua/tardis/interiors/modules/sh_templates.lua

```diff
@@ -88,17 +88,28 @@ function TARDIS:AddInteriorPartsRotation(template, rotate_ang)
 end
 
 
-function TARDIS:MergeInteriorTemplates(cur_metadata, apply_conditions, ent)
-    if not cur_metadata.Templates then
-        return cur_metadata
+function TARDIS:SetupTemplateUpdates(id)
+    local t = self.MetadataRaw[id]
+    if not t.Templates then return end
+
+    for template_id, template in pairs(t.Templates) do
+        if template then
+            self.IntUpdatesPerTemplate[template_id] = self.IntUpdatesPerTemplate[template_id] or {}
+            local int_updates = self.IntUpdatesPerTemplate[template_id]
```

Comment:
No need to set an int_updates variable here you can just set it directly to true e.g. `self.IntUpdatesPerTemplate[template_id][id] = true`

Reply:
this was just a leftover from some previous code, removed in b0f79d581a3c8ce794415eabc8b959a0ab33bfe4